cmd/tuf/import_key.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/flynn/go-docopt"
+	"github.com/theupdateframework/go-tuf"
+	"github.com/theupdateframework/go-tuf/data"
+	"github.com/theupdateframework/go-tuf/pkg/keys"
+)
+
+func init() {
+	register("import-key", cmdImportKey, `
+usage: tuf import-key [--expires=<days>] <role> <file>
```

Comment:
Strong preference for `--role`, I don't believe in commands with >1 positional argument after using `ln` for over a decade and still never getting it right on the first try.

Reply:
Have switched it over to an option argument for `--role` if that's more consistent with the rest of the tool.